neqo-bin/src/bin/client.rs

```diff
@@ -7,6 +7,11 @@
 use clap::Parser as _;
 
 #[tokio::main(flavor = "current_thread")]
+#[allow(
+    clippy::allow_attributes,
+    clippy::unwrap_in_result,
+    reason = "FIXME: False positive?"
```

Comment:
Did you find an issue regarding this?  It's possible that there is a change in the async function desugaring that is tripping these.

Reply:
No, but I only googled for a few minutes.